Resources/Prototypes/Entities/Objects/Fun/Instruments/instruments_misc.yml

```diff
@@ -124,4 +124,54 @@
   - type: Clothing
     sprite: Objects/Fun/Instruments/bike_horn.rsi
     slots: [Belt]
-    quickEquip: false
\ No newline at end of file
+    quickEquip: false
+
+- type: entity
+  parent: PhoneInstrument
+  id: BananaPhone
+  name: Banana Phone
+  description: A direct line to the Honkmother. Seems to always go to voicemail.
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/Instruments/otherinstruments.rsi
+    state: banana_phone
+  - type: EmitSoundOnLand
+    sound:
+       path: /Audio/Items/bikehorn.ogg
+  - type: Instrument
+    program: 124
```

Comment:
Maybe use the gilded bike horn sound or add something unique? Theres a few wacky base midi sounds left, try those out.

Reply:
i will be trying and fixing all these points today and yes the storage was very random therefore experimental at most.

Reply:
will look into new midis